.github/workflows/qa-automation.yml

```diff
@@ -1,39 +1,35 @@
 name: Automate QA
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
`pull_request_target` + `actions/checkout@v3` allows secrets exfiltrating

https://medium.com/tinder/exploiting-github-actions-on-open-source-projects-5d93936d189f

I think we can simply remove the `actions/checkout@v3` step

Reply:
okay, fixed